### PR TITLE
Use LogInHelper from core module

### PR DESCRIPTION
**Pull request by weilu**
Depends on https://github.com/openimis/openimis-be-core_py/pull/313

### CODE REVIEW

**Comment by sniedzielski** [comment]
LGTM, I'll merge once dependent PR is merged (in core module)

**Review by sniedzielski — APPROVED** [review]

**Review by delcroip — APPROVED** [review]

**Comment by weilu** [comment]
Thanks @sniedzielski & @delcroip. I see the CI was triggered before the upstream dependant PR was merged. Can you manaully trigger the CI run & merge if it's good?

**Pull request merged**
